### ISSUE TITLE
System.IO.FileNotFoundException

### ISSUE THREAD

**Issue by bnagajagadeesh** [mihaitibrea/docker-windows-service#3]
I'm getting below in step 9. Is anyone faced this issue? If yes, can you please provide some pointers to resolve the issue?

Step 9/11 : RUN "C:/Windows/Microsoft.NET/Framework64/v4.0.30319/installutil.exe"   /LogToConsole=true /ShowCallStack   C:/app/WindowsContainer.Wrapper.exe ;
 ---> Running in a826ff83064f
Microsoft (R) .NET Framework Installation utility Version 4.8.4084.0
Copyright (C) Microsoft Corporation.  All rights reserved.

Exception occurred while initializing the installation:
System.IO.FileNotFoundException: Could not load file or assembly 'file:///C:\app\WindowsContainer.Wrapper.exe' or one of its dependencies. The system cannot find the file specified..
The command 'powershell -Command $ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue'; "C:/Windows/Microsoft.NET/Framework64/v4.0.30319/installutil.exe"   /LogToConsole=true /ShowCallStack   C:/app/WindowsContainer.Wrapper.exe ;' returned a non-zero code: 1

**Comment by mihaitibrea** [comment]
can you give us more details ? when are you getting this error ?

**Comment by bnagajagadeesh** [comment]
I've cloned the repo, built the solution locally, navigated to docker file diretory and then tried to ran below command
**docker image build --no-cache -t windowsservicewrapper:latest -f .\ServiceWrapper.dockerfile .**

I've got this error when I ran above command and below was the log which has the error

Microsoft Windows [Version 10.0.19042.1237]
(c) Microsoft Corporation. All rights reserved.

C:\Users\vmadmin\source\repos\docker-windows-service\WindowsContainer.Wrapper>docker image build --no-cache -t windowsservicewrapper:latest -f .\ServiceWrapper.dockerfile .
Sending build context to Docker daemon  14.91MB
Step 1/11 : FROM mcr.microsoft.com/dotnet/framework/sdk:4.8 AS build
 ---> 22fb9c46e721
Step 2/11 : WORKDIR /app
 ---> Running in 0cdb24e36522
Removing intermediate container 0cdb24e36522
 ---> 2bbe24898174
Step 3/11 : COPY . .
 ---> 259e1d85d9e5
Step 4/11 : RUN msbuild WindowsContainer.Wrapper.csproj -t:restore -p:Configuration=Release
 ---> Running in 5e2d2d361316
Microsoft (R) Build Engine version 16.11.0+0538acc04 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 9/20/2021 11:29:51 AM.
Project "C:\app\WindowsContainer.Wrapper.csproj" on node 1 (Restore target(s)).
_GetAllRestoreProjectPathItems:
  Determining projects to restore...
Restore:
  Nothing to do. None of the projects specified contain packages to restore.
Done Building Project "C:\app\WindowsContainer.Wrapper.csproj" (Restore target(s)).

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:02.22
Removing intermediate container 5e2d2d361316
 ---> 5d19a9940e16
Step 5/11 : FROM mcr.microsoft.com/dotnet/framework/runtime:4.8
 ---> 0454b67602cd
Step 6/11 : SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 ---> Running in 3f578fcb7922
Removing intermediate container 3f578fcb7922
 ---> a04e0af2316f
Step 7/11 : WORKDIR /app
 ---> Running in 3a8284a13ca2
Removing intermediate container 3a8284a13ca2
 ---> ec0a22a4a107
Step 8/11 : COPY --from=build /app/bin/Release ./
 ---> a9e080402d82
Step 9/11 : RUN "C:/Windows/Microsoft.NET/Framework64/v4.0.30319/installutil.exe"   /LogToConsole=true /ShowCallStack   C:/app/WindowsContainer.Wrapper.exe ;
 ---> Running in 5457fa3c8d73
Microsoft (R) .NET Framework Installation utility Version 4.8.4084.0
Copyright (C) Microsoft Corporation.  All rights reserved.

**Exception occurred while initializing the installation:
System.IO.FileNotFoundException: Could not load file or assembly 'file:///C:\app\WindowsContainer.Wrapper.exe' or one of its dependencies. The system cannot find the file specified..
The command 'powershell -Command $ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue'; "C:/Windows/Microsoft.NET/Framework64/v4.0.30319/installutil.exe"   /LogToConsole=true /ShowCallStack   C:/app/WindowsContainer.Wrapper.exe ;' returned a non-zero code: 1**
